tests/functional/settings_secondary_emails.js

```diff
@@ -77,6 +77,8 @@ define([
         .then(click('.email-add:not(.disabled)'))
         .then(testElementExists('.not-verified'))
         .then(click('.email-address .settings-button.warning.email-disconnect'))
+        .then(visibleByQSA('.success'))
+        .then(click('#emails .settings-unit-stub button'))
```

Comment:
@shane-tomlinson This test consistently fails in circle-ci but not locally,

```
The element reference of <button class="settings-button settings-unit-toggle hidden secondary"> stale: either the element is no longer attached to the DOM or the page has been refreshed
```

At a glance, is there anything that jumps out that I am not doing with the test?

Reply:
@vbudhram After disconnect, is the panel re-rendered? If so then there will be 2 of those elements matching the selector, the old one that got removed and the new rendered one